### PR TITLE
Feature/interpretations improvements

### PR DESCRIPTION
**Pull request by tokland**

### CODE REVIEW

**Comment by adrianq** [comment]
@tokland is this ready to be merged?

File: dhis-2/dhis-web/dhis-web-dashboard-integration/src/main/webapp/dhis-web-dashboard-integration/dashboard.vm [hunk 1]
```diff
@@ -103,6 +103,11 @@ jQuery( document ).ready( function() {
     <input type="button" class="interpretationButton" value="$i18n.getString( 'share' )" onclick="dhis2.db.shareInterpretation()">
 </div>
 
+<div id="shareFormError">
+    <p>$i18n.getString( 'cannot_write_interpretation' )</p>
+    <input type="button" class="interpretationButton" value="$i18n.getString( 'close' )" onclick="$(this).closest('.ui-dialog-content').dialog('close');">
```
Comment (adrianq) [inline dhis-2/dhis-web/dhis-web-dashboard-integration/src/main/webapp/dhis-web-dashboard-integration/dashboard.vm]: To be coherent with the look & feel. Can we use something like
`<a class="greyButtonLink" href="javascript:dhis2.db.addDashboard()" style="margin-top: 1px">$i18n.getString( "create" )</a>` ?
Extracted from a few lines below..

Comment (adrianq) [inline dhis-2/dhis-web/dhis-web-dashboard-integration/src/main/webapp/dhis-web-dashboard-integration/dashboard.vm]: Obviously changing literals and anything  that needs to be changed

File: dhis-2/dhis-web/dhis-web-dashboard-integration/src/main/webapp/dhis-web-dashboard-integration/javascript/dashboard.js [hunk 2]
```diff
@@ -1429,19 +1431,27 @@ dhis2.db.registerDashboardViewEvent = function () {
...
+            $(selector).dialog({
+                modal: true,
+                width: 550,
+                resizable: false,
+                title: title
+            });
+        }
     });
 }
-
```
Comment (adrianq) [inline dhis-2/dhis-web/dhis-web-dashboard-integration/src/main/webapp/dhis-web-dashboard-integration/javascript/dashboard.js]: Do not remove this break line

**Comment by tokland** [comment]
Review proposal commited.

**Pull request merged**